adage/adage/settings.py

```diff
@@ -23,10 +24,17 @@
     CONFIG = CI_CONFIG
     pg_user = os.environ.get('PG_USER')
     pg_pass = os.environ.get('PG_PASSWORD')
-    CONFIG['dbmaster']['USER'] = pg_user
-    CONFIG['dbmaster']['PASSWORD'] = pg_pass
     CONFIG['databases']['default']['USER'] = pg_user
     CONFIG['databases']['default']['PASSWORD'] = pg_pass
+elif os.environ.get('CIRCLECI') == 'true':
+    with open(os.path.join(BASE_DIR, 'adage', 'config.py.template')) as f:
+        exec f
+    CONFIG = CI_CONFIG
+    CONFIG['databases']['default']['NAME'] = 'circleci_test'
```

Comment:
I called it `CI_CONFIG` but I might have been too optimistic about generalizing. It is really `CODESHIP_CONFIG`, I guess, since CircleCI seems to do things differently. Maybe we should rename it that way to avoid confusion and create a `CIRCLECI_CONFIG` that is separate. Will we be using both Codeship and CircleCI going forward?